searx/engines/reddit.py

```diff
@@ -30,6 +31,9 @@ def request(query, params):
                        'limit': page_size})
     params['url'] = search_url.format(query=query)
 
+    # using searx User-Agent
+    params['headers']['User-Agent'] = searx_useragent()
```

Comment:
User-agent appears in their logs, so it can reduce anonymity. I think the original random user-agent would be better here.